src/ConsoleApp/consoleAppDemo.java

```diff
@@ -6,16 +6,91 @@
 
 public class consoleAppDemo {
 	
-
+	static void response () {
+		System.out.println("not yet Implimented");
+		return;
+	}
+	
+	static void error () {
+		System.out.println("Please Enter Valid Option");
+		return;
+	}
+	
+	static void endprogram() {
+		System.out.println("CLosing Application...");
+	}
+	
 	public static void main(String[] args) {
-
-		System.out.println("Brewmaster");
 		
+		String input;
+		int choice = 0;
+		
+		Scanner scan = new Scanner(System.in);
+		
+		System.out.println("Brewmaster");
 		System.out.println("Press enter to continue");
+		
+		input = scan.nextLine();
+		
+		System.out.println("Press 1 for open Tavern Owners Menu");
+		System.out.println("Press 2 for view Inventory");
+		System.out.println("Press 3 for Admin Menu");
+		System.out.println("press 4 to Close Application");
+		System.out.println("");
+		System.out.println("ENTER your option -> :");
 
-		Scanner in = new Scanner(System.in);
-		in.next();
```

Comment:
This code is repeated 5 times. You only need it once. See this pseudo-code;
`boolean closeApplication = false;
while(!closeApplication){
    print the menu;
    take input;
    switch statement (does not reprint menu);
    if option 4, set closeApplication to true;}`